robot_descriptions/aloha_mj_description.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""ALOHA 2 MJCF description."""
+
+from os import getenv as _getenv
+from os import path as _path
+
+from ._cache import clone_to_cache as _clone_to_cache
+
+REPOSITORY_PATH: str = _clone_to_cache(
+    "mujoco_menagerie",
+    commit=_getenv("ROBOT_DESCRIPTION_COMMIT", None),
+)
+
+PACKAGE_PATH: str = _path.join(REPOSITORY_PATH, "aloha")
+
+MJCF_PATH: str = _path.join(PACKAGE_PATH, "scene.xml")
```

Comment:
Not found according to the CI:

```
2024-03-26T09:30:39.9809515Z ValueError: mjParseXML: resource not found via provider or OS filesystem: '/home/runner/.cache/robot_descriptions/mujoco_menagerie/aloha/scene.xml'
```

I guess it's a newer model and we should update the Menagerie commit in `_repositories.py` as well?